packages/core/stories/elements/Button/Button.stories.mdx

```diff
@@ -12,160 +12,153 @@ import * as stories from './Button.stories.js';
 />
 
 # Buttons
-Buttons express what action will occur when the user clicks or touches it. Buttons are used to initialize an action, either in the background or foreground of an experience.
-<Canvas withSource="none" >
+
+Buttons are used in forms and interactive applications, but also as prominent call-to-action links on website pages.
+
+<Canvas withSource="none">
   <Story
-    name='test'
-    parameters={{ docs: { source: {code: null}}}}>
-    <Button text="Button" />
-    </Story>
+		name="button-primary"
+    parameters={{ docs: { transformSource: () => renderReactMarkup(<Button usage="" text="Button"/>) } }}>
+      <Button usage="" text="Button" />
+  </Story>
 </Canvas>
 
-## Overview
-Buttons are vital for creating a smooth conversational flow on the web, and should be used sparingly. Buttons should adhere to standard web practices, and afford for a target area of 45 pixels. They should also appear slightly raised, and not be completely flat. They have a slight shadow on them so they appear clickable. The small buttons are often used on forms and to set filters on listing pages. The large buttons should be used for primary calls to action. Font sizes reduce slightly on mobile.
+Following web best practices buttons have a target area of 45 pixels. To provide more visual affordance that they are clickable—not just text in a box—they have a shadow to appear slightly raised. Small buttons can be used for secondary tasks on forms—for example image upload—and filters on listing pages. Large buttons should be used for form submits or primary calls to action. There should usually be only one primary button in any UI. Font sizes reduce slightly on mobile.
 
+---
 
-## Variations
-<br />
+### Primary Buttons
 
-### Primary Button
-**Usage:** For the principle call to action on the page.
+- **Usage:** For the main action in a form, or for a call-to-action link on a page.
+- **Style:** Default buttons are solid color with contrasting uppercase text. For actions with text longer than two words use the `title-case` class. On hover the Background color becomes lighter.
```

Comment:
so the "title-case" option is not currently available to Button, it is though in ButtonWithIcon in react (but not in patternlab), which we should make available to all buttons. These are the things I was talking about that we need to follow up on refactoring and consolidating in code. 
```suggestion
- **Style:** Default buttons are solid color with contrasting uppercase text. For actions with text longer than two words use the title case instead of all caps. On hover, the Background color becomes lighter.
```

Reply:
for now, a class is not available, so let's just describe the rule (and add that as a to-do in code)